resources.go

```diff
@@ -64,6 +68,7 @@ func Provider() tfbridge.ProviderInfo {
 		License:              "Apache-2.0",
 		Homepage:             "https://pulumi.io",
 		Repository:           "https://github.com/pulumi/pulumi-kafka",
+		TFProviderLicense:    tfLicenseTypeRef(tfbridge.MITLicenseType),
```

Comment:
Can you just do `&tfbridge.MITLicenseType` here and dispense with the function?

Reply:
I can't use the address of an Enum value so the func will pass the enum value and take the address of the underlying type, `tfbridge.TFProviderLicense`